operators/multiclusterobservability/pkg/rendering/renderer_mcoa.go

```diff
@@ -212,6 +341,14 @@ func (r *MCORenderer) renderAddonDeploymentConfig(
 			}
 		}
 
+		if (cs.Platform != nil && cs.Platform.Metrics.Collection.Enabled) || (cs.UserWorkloads != nil && cs.UserWorkloads.Metrics.Collection.Enabled) {
+			obsAPIURL, err := mcoconfig.GetObsAPIExternalURL(context.TODO(), r.kubeClient, namespace)
```

Comment:
On the value itself I'm still not sure about the use of the key `signalsHubEndpoint` to represent it, mainly because this is still a very metrics oriented value. I'm thinking if we could split it so that on the short term would be more useful to more signals.

Taking the following two endpoints on the hub: 
- `https://obs-api-open-cluster-management-observability.apps.myhubcom/api/metrics/v1`
- `https://mcoa-managed-instance-openshift-logging.apps.myhub.foo.com/api/logs/v1/cluster-1/otlp/v1/logs`

Then:
- `hubHostname` (would be there if any Managed Storage is enabled): `apps.myhub.foo` or `myhub.foo`
- `hubMetricsRoute`: `obs-api-open-cluster-management-observability`

WDYT? I'm suggesting something like this because then logs&traces would be able to re-use `hubHostname`. Regardless if we do unified OBS Api or separated ones. (This could also be addressed in a follow up)